awx/ui_next/README.md

```diff
@@ -0,0 +1,39 @@
+# Instruction to build ui_next directly from this directory
+
+## Set src of the ui_next repo
+
+### via GIT
```

Comment:
Since the Makefile will use the internal repo could we say something like `from a custom GIT repo`

Reply:
Also, it looks like a bad find and replace happened. This should be `UI_NEXT_GIT_REPO` and the branch should just be `UI_NEXT_GIT_BRANCH`

Reply:
fixed